### ISSUE TITLE
Formula for CBMC 5.69.1 is missing

### ISSUE THREAD

**Issue by adpaco-aws** [diffblue/cbmc#7286]
The formula for CBMC 5.69.1 is missing, so we cannot install CBMC through `brew` on macOS platforms. This is blocking our upgrade to CBMC 5.69.1 in https://github.com/model-checking/kani/pull/1811

**Comment by NlightNFotis** [comment]
Hello @adpaco-aws @feliperodri 

Our apologies for this - we had a CI failure in our 5.69.0 release, so we had to make a point release on an accelerated schedule, which caused us to miss a step in our usual release process.

I have now updated the tap to contain the [formula](https://github.com/diffblue/homebrew-cbmc/commit/323400e425388bb370726a4e3ed1d4aa88848fa0) and the binaries for `5.69.1`.

Let me know if there's anything else we can do to help.
